build.sbt

```diff
@@ -15,20 +15,19 @@ scalacOptions := Seq(
 )
 
 libraryDependencies ++= Seq(
-  "com.azavea"                  %% "scaliper"              % "0.5.0-SNAPSHOT" % "test",
   "io.dylemma"                  %% "xml-spac"              % "0.3",
-  "org.apache.spark"            %% "spark-hive"            % "2.2.0" % "provided",
   "org.apache.hadoop"           %  "hadoop-aws"            % "2.8.1"
     exclude("javax.servlet",     "servlet-api")
     exclude("javax.servlet.jsp", "jsp-api")
     exclude("org.mortbay.jetty", "servlet-api"),
+  "org.apache.spark"            %% "spark-hive"            % "2.2.0" % "provided",
   "org.locationtech.geotrellis" %% "geotrellis-spark"      % "1.2.0-SNAPSHOT",
   "org.locationtech.geotrellis" %% "geotrellis-util"       % "1.2.0-SNAPSHOT",
   "org.locationtech.geotrellis" %% "geotrellis-vector"     % "1.2.0-SNAPSHOT",
   "org.locationtech.geotrellis" %% "geotrellis-vectortile" % "1.2.0-SNAPSHOT",
-  "org.scalatest"               %% "scalatest"             % "2.2.0" % "test",
-  "org.typelevel"               %% "cats"                  % "0.9.0",
-  "org.spire-math"              %% "spire"                 % "0.13.0"
+  "org.scalatest"               %% "scalatest"             % "3.0.1" % "test",
+  "org.spire-math"              %% "spire"                 % "0.13.0",
+  "org.typelevel"               %% "cats-core"             % "1.0.0-MF"
```

Comment:
What is MF? Is this an release?

Reply:
"Milestone Final" by their wording.

Reply:
It's the version they have displayed as official on their [microsite](https://typelevel.org/cats/).

Reply:
:+1: